apps/authoring/src/renderer/models/preferences/model-preferences-hooks.ts

```diff
@@ -18,9 +19,11 @@ export const useProcessing = () => {
 export const useOpen = (to = '/settings/theme') => {
   const navigator = useNavigate();
 
-  Menu.File.onPreferencesOpen(() => {
-    navigator(to);
-  });
+  useEffect(() => {
```

Comment:
In components that re-render often based on state change, we need to register listeners using useEffect and/or remove the listener before the next re-render to avoid memory leak due to multiple listeners added to the same emitter. 

This is the error: `MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 'listener' listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit...`